app.py

```diff
@@ -39,36 +39,25 @@
     cdk_app, f"{stack_name_prefix}-load-balancer", network_stack.vpc
 )
 
-apex_service_props = ServiceProps(
-    "schematic-apex",
-    8000,
-    200,
-    f"ghcr.io/sage-bionetworks/schematic-apex:{image_version}",
-    {
-        "API_DOCS_HOST": "schematic-api-docs",
-        "API_DOCS_PORT": "8010",
-        "API_GATEWAY_HOST": "schematic-api-gateway",
-        "API_GATEWAY_PORT": "8082",
-        "APP_HOST": "schematic-app",
-        "APP_PORT": "4200",
-        "THUMBOR_HOST": "schematic-thumbor",
-        "THUMBOR_PORT": "8889",
-        "ZIPKIN_HOST": "schematic-zipkin",
-        "ZIPKIN_PORT": "9411",
-    },
+app_service_props = ServiceProps(
+    "schematic-app",
+    443,
+    1024,
+    "ghcr.io/sage-bionetworks/schematic:v0.1.90-beta",
+    {},
+    "schematic-dev-DockerFargateStack/dev/ecs",
 )
```

Comment:
The image and the location of the secret will need to be extracted out to `cdk.json` so we can run different values per environment

Reply:
this is now parameterized with the environment